### PR TITLE
[TUBEMQ-439] Add cluster info query

### PR DESCRIPTION
**Pull request by EMsnap**
[TUBEMQ-439] Add cluster info query

### CODE REVIEW

**Review by gosonzhang — CHANGES_REQUESTED** [review]
1. Version information missing
2. TubeMQ, not tube

File: tubemq-manager/src/test/java/org/apache/tubemq/manager/controller/TestNodeController.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        return nodeEntry;
+    }
+
+    @Test
+    public void testClusterInfo() throws Exception {
+        RequestBuilder request = get("/v1/node/query?method=admin_query_cluster_info&" +
+                "type=op_query&clusterId=1");
+        MvcResult result = mockMvc.perform(request).andReturn();
+        String resultStr = result.getResponse().getContentAsString();
+        log.info("result json string is {}, response type is {}", resultStr,
```
Comment (yuanboliu) [inline tubemq-manager/src/test/java/org/apache/tubemq/manager/controller/TestNodeController.java:71]: please use Assert.equals to make sure the result value is correct.

**Comment by yuanboliu** [comment]
+1 pending on travis result

**Review by gosonzhang — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]
+1

**Pull request merged**